LICENSE

```diff
@@ -200,3 +200,30 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 
+----
+
+This product bundles Glyphicon Halflings ( http://glyphicons.com/
+ ) via Bootstrap 3's distributing these under The MIT License.
+
+This product bundles angular-ui / ui-sortable / sortable.js ( https://github.com/angular-ui/ui-sortable/blob/master/src/sortable.js ) under The MIT License.
+
+
+The MIT License (MIT)
```

Comment:
hmm :thinking: 
well better safe than sorry :+1:
thanks for looking into that @apetro!